### DISCUSSION TITLE
New Terminal Prompt (2.9.7.1)

### DISCUSSION THREAD

**Discussion by adityapandeydev** [mylinuxforwork/dotfiles#529] [Ideas]
Just installed the 2.9.7.1 version to checkout. I don't know if its already planned or not, but it would be great if the theme of the new terminal prompt also matched the overall theme, because right now no matter what wallpaper i use the terminal prompt is same black and grey.

**Comment by mylinuxforwork** [comment]
Hi. I have already opened a request on oh-my-posh how to integrate pywal. For now it’s a neutral gray until I find a solution.
